### PR TITLE
Begin Global Stylesheet

### PR DESCRIPTION
**Pull request by jusoakley**
See [SCRUM-22] for details.

This is a good start, I think, though typography is obviously missing for now. The I tried to add lots of classes to eliminate the need for writing repetitive component styles.

There are a few exceptions (i.e CSS :not()) here that are only in place to not break other elements that don't have designs yet, like the name input.

Also, there are some pretty obvious conflicts with rebasing on master that we'll resolve eventually.

[SCRUM-22]: https://gamepals.atlassian.net/browse/SCRUM-22

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ services:
       - "5433:5432" # order is host:container
     logging:
       driver: "none"
-    volumes:
-      - ./docker/postgres-data:/var/lib/postgresql/data
+    # volumes:
```
Comment (DannyPinghero) [inline docker-compose.yml:23]:
```suggestion
    volumes:
```

File: docker-compose.yml [hunk 2]
```diff
@@ -20,8 +20,8 @@ services:
...
+    #   - ./docker/postgres-data:/var/lib/postgresql/data
```
Comment (DannyPinghero) [inline docker-compose.yml:24]:
```suggestion
    - ./docker/postgres-data:/var/lib/postgresql/data
```

Comment (DannyPinghero) [reply docker-compose.yml:24]: cant comment these out in the repo - ill show you how to set up an override file that will go in the gitignore

Comment (jusoakley) [reply docker-compose.yml:24]: Yeah this was an oversight on my part, I never meant to add them in the first place.

**Comment by jusoakley** [comment]
> see comment about the docker-compose
> 
> Also let me know if you would like to clean the frontend test failure [ https://travis-ci.org/github/GamePalsDoinCode/scrumsistant/jobs/705505984 ]. The problem is the test was selecting a button in a hacky way which now fails with the cleaned up css.
> 
> lines are
> 
> ```
> const logoutButton = fixture.debugElement.query(
>       By.css('div.ceremonyContainer > button:nth-child(4)')
>     ) // obv this is a hack shortcut while I wait for someone to style the page
> ```
> 
> in `dashboard.component.spec.ts`

I can give that a shot, this selector syntax looks reasonable.

**Comment by jusoakley** [comment]
ok looks like I actually don't know what I'm doing.

**Comment by jusoakley** [comment]
Nope turns out I know nothing about frontend testing.

**Comment by jusoakley** [comment]
I squashed on my local to make the history cleaner and also to remove my shame but I can't push it (protected hook declined error again) so yeah here's this disgusting trash history that passes the tests.

File: frontend/src/app/dashboard/dashboard.component.spec.ts [hunk 3]
```diff
@@ -101,7 +101,7 @@ describe('DashboardComponent', () => {
   }))
   it('should call auth logout when logout button is clicked', fakeAsync(() => {
     const logoutButton = fixture.debugElement.query(
-      By.css('div.ceremonyContainer > button:nth-child(4)')
+      By.css('.ceremonyContainer .logout')
     ) // obv this is a hack shortcut while I wait for someone to style the page
```
Comment (DannyPinghero) [inline frontend/src/app/dashboard/dashboard.component.spec.ts:105]:
```suggestion
    )
```

**Comment by DannyPinghero** [comment]
> I squashed on my local to make the history cleaner and also to remove my shame but I can't push it (protected hook declined error again) so yeah here's this disgusting trash history that passes the tests.

haha gets squashed in the merge anyway so you gravy

**Pull request merged**